### PR TITLE
Resolving some issues

### PR DESCRIPTION
**Pull request by adrigzr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,25 +34,27 @@ Installation
...
   importScriptsGlobPatterns: [
     'assets/service-workers/*.js'
   ]
 };
 ```
 
+```javascript
+// ember-cli-build.js
+const app = new EmberAddon(defaults, {
+  importScriptsTransform(importScripts) {
```
Comment (josex2r) [inline README.md:52]: why not having a `workbox` property instead of defining the function in the root object? Something like:

```js
const app = new EmberAddon(defaults, {
  workbox: {
    importScriptsTransform
  }
});
```

File: addon/services/service-worker.js [hunk 2]
```diff
@@ -43,8 +50,14 @@ export default class ServiceWorker extends Service.extend(Evented) {
 	}
 
 	async register(swFile) {
+		let path = `${this.config.rootURL}${swFile}`;
+
+		if (path[0] === '/') {
```
Comment (josex2r) [inline addon/services/service-worker.js:55]: Is this change necessary??

```js
// http://localhost:4200/index.html
navigator.serviceWorker.register('/sw.js')
// registers https://localhost:4200/sw.js


// http://localhost:4200/foo/bar/index.html
navigator.serviceWorker.register('/sw.js')
// registers https://localhost:4200/sw.js
```

**Review by josex2r — APPROVED** [review]

**Review by xe21500 — APPROVED** [review]

**Review by cmatilla — APPROVED** [review]
